GitCommands/Submodules/SubmoduleStatusProvider.cs

```diff
@@ -139,7 +140,7 @@ public void UpdateSubmodulesStatus(string workingDirectory, [CanBeNull] IReadOnl
 
             // Throttle updates triggered from status updates
             TimeSpan elapsed = DateTime.Now - _previousSubmoduleUpdateTime;
-            if (gitStatus == null || elapsed.TotalSeconds <= 15)
+            if (gitStatus == null || (!forceUpdate && elapsed.TotalSeconds <= 15))
```

Comment:
💭  I'm curious how this is supposed to work. If a request arrives during the delay period, is it scheduled for later execution or simply ignored?

Reply:
Valid point in general but not for this PR. It seems to be simply ignored.
Though there's a cyclic update after 5 minutes IIRC. Am I right, @gerhardol?

Reply:
Fast updates are just ignored for submodules, so the displayed status may be incorrect here.
For the Commit-status (in button and tool tip for artificial commits), the request is done with a delay as well as periodically.
A submodule status update could be much more expensive than a git-status (that still could need seconds).

In 2.x, the status was refreshed when the form was activated, so it ran much more often. (Pulldown of the menu often failed for me as most repos I work with have many submodules.)